src/mmw/js/src/user/views.js

```diff
@@ -279,8 +281,93 @@ var ForgotModalView = Marionette.ItemView.extend({
     }
 });
 
+var ItsiRegisterModalView = Marionette.ItemView.extend({
```

Comment:
And this model property gets overwritten with the actual model when instantiating the view? I can imagine that being a little confusing when looking at the code. If all we want to do is annotate it, then we could do it in a comment.